README.md

```diff
@@ -1,24 +1,226 @@
-# README
+<a name="readme-top"></a>
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="Kreative-Keeper.png" alt="logo" width="auto"  height="auto" />
+  <br/>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+  <h3><b>Microverse README Template</b></h3>
 
-Things you may want to cover:
+</div>
 
-* Ruby version
+<!-- TABLE OF CONTENTS -->
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 [Kreative Keeper] <a name="about-project"></a>
 
-* How to run the test suite
+**[Kreative Keeper]** is a fully functional blog website that shows list of posts and empowers readers to interact with them by adding comments and liking posts
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+> Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
# [OPTIONAL]

- [ ] Kindly remove this text to  make your readme file clean

Reply:
Done